test/e2e/domainmapping/domain_mapping_test.go

```diff
@@ -83,6 +83,9 @@ func TestBYOCertificate(t *testing.T) {
 	secret, err := clients.KubeClient.CoreV1().Secrets(test.ServingFlags.TestNamespace).Create(ctx, &corev1.Secret{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: test.AppendRandomString("byocert-secret"),
+			Labels: map[string]string{
```

Comment:
So the current user, who is using BYO cert for DomainMapping, must add the label to every single secrets?
https://github.com/knative-sandbox/net-istio/pull/920 will be a breaking change, won't it?

Reply:
If it is a secret that someone creates manually yes. For the ones we copy from the user ns to the istio one no.
For the ones created by cert manager we will use the secret template available for cm certificates to inject automatically that label selector.

Reply:
> If it is a secret that someone creates manually yes. 

Hmm... BYO cert for DomainMapping has been Beta stage and some users are already using them. We will take care of the users by documentation only? Or there is any good solution we can offer?